src/printer.js

```diff
@@ -376,7 +376,7 @@ function printStatement(path, options, print) {
           ";",
           concat(
             path.map(
-              usegroup => concat([hardline, print(usegroup), ";"]),
+              usegroup => concat([hardline, print(usegroup)]),
```

Comment:
Shall we rename `usegroup` to `child` here? I've been confused about this because i didn't realize that the children of the namespaces can be not only usegroups, but also classes...

Reply:
@czosel yep,
```php
namespace My {
 // Here we can have any stuff
}
```
Let's do this.

Reply:
@czosel but parser always interpreted as `usegroup`, let wait what says @mgrip

Reply:
Are you sure? In my debugger ist just looked like in the test case there were 3 `usegroup`s and one `class` in `children`. Since we're just calling `print` on the children, the parser doesn't care what kind of node it finds.

Reply:
@czosel let's double check :+1: